### ISSUE TITLE
PackingList element in SMPTE Assetmap

### ISSUE THREAD

**Issue by matmat** [wolfgangw/backports#63]
The definition in SMPTE-429-9-2007-AM.xsd is:
 
<xs:element name="PackingList" type="xs:boolean" minOccurs="0"/>

although dcp_inspect gives an error when

<PackingList>false</PackingList>

occurs an SMPTE assetmap:

E.g.
Error: AM 23945607-ec4d-4c57-8b6e-fda4971a8400: SMPTE AM requires a PackingList element to contain the value 'true' (value is missing). Continuing anyway
Error: Not a PackingList: Blodssystrar_FTR-1_F_SV-SV_51_2K_20150925_JMF_SMPTE/36fda57a-775a-409b-98da-59609afdb2f2_j2c.mxf

This Assetmap validates against the SMPTE-429-9-2007-AM.xsd schema:

$ xmllint --nonet --noout --schema SMPTE-429-9-2007-AM.xsd Blodssystrar_FTR-1_F_SV-SV_51_2K_20150925_JMF_SMPTE/ASSETMAP.xml
Blodssystrar_FTR-1_F_SV-SV_51_2K_20150925_JMF_SMPTE/ASSETMAP.xml validates

**Comment by wolfgangw** [comment]
You are right, according to SMPTE-429-9-2007-AM.xsd the PackingList element with either true or false value is allowed in any Asset element.

Will change to hint class message.
